applications/DEMApplication/custom_constitutive/DEM_rolling_friction_model_constant_torque.cpp

```diff
@@ -29,56 +29,50 @@ namespace Kratos{
+            // Get rolling friction coefficient
+            Properties& properties_of_this_contact = p_element->GetProperties().GetSubProperties(p_neighbor->GetProperties().Id());
+            const double rolling_friction_coefficient = properties_of_this_contact[ROLLING_FRICTION];
 
-        } 
+            mContactMoment[0] -= elementRelAngularVelocity_normalise[0] * fabs(LocalContactForce[2]) * equiv_radius * rolling_friction_coefficient; 
+            mContactMoment[1] -= elementRelAngularVelocity_normalise[1] * fabs(LocalContactForce[2]) * equiv_radius * rolling_friction_coefficient; 
+            mContactMoment[2] -= elementRelAngularVelocity_normalise[2] * fabs(LocalContactForce[2]) * equiv_radius * rolling_friction_coefficient; 
+        }
     }
 
-    void DEMRollingFrictionModelConstantTorque::ComputeRollingFrictionWithWall(double LocalContactForce[3], SphericParticle* p_element, Condition* const wall, double indentation, array_1d<double, 3>& mContactMoment)
+    void DEMRollingFrictionModelConstantTorque::ComputeRollingFrictionWithWall(SphericParticle* p_element, Condition* const wall, double LocalCoordSystem_2[3], double LocalContactForce[3], array_1d<double, 3>& mContactMoment)
     {
         array_1d<double, 3> element1AngularVelocity;
         noalias(element1AngularVelocity) = p_element->GetGeometry()[0].FastGetSolutionStepValue(ANGULAR_VELOCITY);
         if (element1AngularVelocity[0] || element1AngularVelocity[1] || element1AngularVelocity[2]){
-
-            const double my_young      = p_element->GetYoung(); 
-            const double walls_young   = wall->GetProperties()[YOUNG_MODULUS];
-            
-            double arm_length = p_element->GetInteractionRadius() - indentation* walls_young / (walls_young + my_young); 
-            
+            // Normalize angular velocity
             GeometryFunctions::normalize(element1AngularVelocity);
-
             array_1d<double, 3> element1AngularVelocity_normalise = element1AngularVelocity;
 
-            Properties& properties_of_this_contact = p_element->GetProperties().GetSubProperties(wall->GetProperties().Id());
-
-            mContactMoment[0] -= element1AngularVelocity_normalise[0] * fabs(LocalContactForce[2]) * arm_length * properties_of_this_contact[ROLLING_FRICTION]; 
+            // Get particle radius
+            const double radius = p_element->GetRadius();
 
-            mContactMoment[1] -= element1AngularVelocity_normalise[1] * fabs(LocalContactForce[2]) * arm_length * properties_of_this_contact[ROLLING_FRICTION]; 
-
-            mContactMoment[2] -= element1AngularVelocity_normalise[2] * fabs(LocalContactForce[2]) * arm_length * properties_of_this_contact[ROLLING_FRICTION]; 
+            // Get rolling friction coefficient
+            Properties& properties_of_this_contact = p_element->GetProperties().GetSubProperties(wall->GetProperties().Id());
```

Comment:
r_properties

Reply:
Changed all properties_of_this_contact to r_properties